src/main/java/com/wanted/notion/dto/page/PageResponseDto.java

```diff
@@ -0,0 +1,30 @@
+package com.wanted.notion.dto.page;
+
+import lombok.Builder;
+import lombok.Data;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Data
+@Builder
+public class PageResponseDto {
+    private int pageId;
```

Comment:
dto 생성시에 불변클래스로 생성하는 것이 좋습니다
```suggestion
    private final int pageId;
```